src/img-optm.cls.php

```diff
@@ -1124,7 +1134,7 @@ public function pull($manual = false)
 				$total_pulled_ori++;
 			};
 
-			if (class_exists('Requests')) {
+			if (class_exists('\WpOrg\Requests\Requests') && class_exists('\WpOrg\Requests\Autoload') && !version_compare(PHP_VERSION, '5.6.0', '<')) {
```

Comment:
For `!version_compare(PHP_VERSION, '5.6.0', '<')`, why not use `version_compare(PHP_VERSION, '5.6.0', '>=')`

Reply:
Done~